client/components/isolated-editor/client/components/block-editor-toolbar/more-menu/writing-menu.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * WordPress dependencies
+ */
+import { MenuGroup } from '@wordpress/components';
+import { __, _x } from '@wordpress/i18n';
+import { useSelect } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import FeatureToggle from '../toggle-feature';
+import OptionToggle from '../toggle-option';
+
+/** @typedef {import('../../../index').BlockEditorSettings} BlockEditorSettings */
+/** @typedef {import('./index').OnClose} OnClose */
+
+/**
+ * Writing menu
+ *
+ * @param {object} props - Component props
+ * @param {OnClose} props.onClose - Close the menu
+ * @param {BlockEditorSettings} props.settings - Settings
+ */
+function WritingMenu( { onClose, settings } ) {
+	const { preview, fullscreen, topToolbar } = settings.iso.moreMenu;
+	const { isFullscreen } = useSelect(
+		( select ) => ( {
+			isFullscreen: select( 'isolated/editor' ).isOptionActive( 'fullscreenMode' ),
+		} ),
+		[]
+	);
+
+	// Anything to show?
+	if ( ! fullscreen && ! preview && ! topToolbar ) {
+		return null;
+	}
+
+	return (
+		<MenuGroup label={ _x( 'View', 'noun' ) }>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **38** times:
`translate( 'View', 'Views', { context: 'noun'} )` *ES Score: 13*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/try/isolated-block-editor)


:information_source: This string already exists with the following **contexts**:
- `null` (no context)
- `Stats: List item action to view content`
- `TinyMCE menu`
- `verb`
- `view a subscription`

 Would it make sense to reuse one of the above?